### PR TITLE
Fixes #10690: Bad links, typo, images in french in intro chapter

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/10690

### CODE REVIEW

File: 00_introduction/40_architecture_and_dependencies.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [[architecture]]
 === Technical architecture and software dependencies
 
-==== Functionnal architecture of Rudder
+==== functional architecture of Rudder
```
Comment (amousset) [inline 00_introduction/40_architecture_and_dependencies.txt]: Functional

**Comment by fanf** [comment]
PR rebased

File: 00_introduction/20_key_features.txt [hunk 2]
```diff
@@ -71,13 +73,13 @@ used to defined coniguration rule targets. Typically, some configurations are
 linked to the kind of server (physical or virtual), the quantity of RAM
 available, the version of an OS library which contains a security bug, etc.
 
-All of these data are also available throught Rudder APIs (see link[TOTO]).
+All of these data are also available <<rudder-api-integration, throught Rudder APIs>>.
```
Comment (jooooooon) [inline 00_introduction/20_key_features.txt]: s/throught/through/

File: 00_introduction/20_key_features.txt [hunk 3]
```diff
@@ -71,13 +73,13 @@ used to defined coniguration rule targets. Typically, some configurations are
...
 
 ==== REST API
 
 All Rudder commands are available throught an exhaustive REST API. That API is
```
Comment (jooooooon) [inline 00_introduction/20_key_features.txt]: same here

File: 00_introduction/20_key_features.txt [hunk 4]
```diff
@@ -71,13 +73,13 @@ used to defined coniguration rule targets. Typically, some configurations are
...
 http://www.rudder-project.org/rudder-api-doc/[fully documented online] and can
-be used to todo_link[quickly and smoothly integrate Rudder with you existing infrastructure].
+be used to <<rudder-api-integration, quickly and smoothly integrate Rudder with you existing infrastructure>>.
```
Comment (jooooooon) [inline 00_introduction/20_key_features.txt]: you*r* existing infra

**Pull request merged**